src/veh_interact.cpp

```diff
@@ -1142,9 +1106,14 @@ bool veh_interact::can_remove_part(int veh_part_index, int mech_skill, int msg_w
         if (g->u.has_trait("DEBUG_HS")) {
             return true;
         }
+
+        if( is_wheel && !( g->u.can_lift( *veh ) || has_jack ) ) {
```

Comment:
You're checking `g->u.can_lift( *veh ) || has_jack` quite a bit, you could extract it into a separate variable for readability.